src/server/controllers/users.js

```diff
@@ -128,5 +134,82 @@ module.exports = {
         errorLogs.catchErrors(error);
         res.status(400).send(error);
       });
+  },
+  inviteUser(req, res) {
+    const name = getUsernameFromEmail(req.body.email);
+    req.body.username = name.first + ' ' + name.last;
```

Comment:
I will prefer using `res.locals` to store any variables local to the request. [Link](https://expressjs.com/en/api.html#res.locals)